components/brave_wallet_ui/page/screens/onboarding/welcome/onboarding-welcome.tsx

```diff
@@ -67,69 +67,46 @@ export const OnboardingWelcome = () => {
   }, [setupStillInProgress, report, dispatch])
 
   return (
-    <WalletPageLayout>
-      <OnboardingWrapper>
-        <WalletWelcomeGraphic />
-
-        <Title maxWidth='467px'>{getLocale('braveWalletWelcomeTitle')}</Title>
-
-        <BlockQuote>
-          <VerticalRule />
-          <BlockQuoteTextContainer>
-            <span>{getLocale('braveWalletPerksTokens')}</span>
-            <span>{getLocale('braveWalletMultiChain')}</span>
-            <span>{getLocale('braveWalletPerksBrowserNative')}</span>
-          </BlockQuoteTextContainer>
-        </BlockQuote>
-
-        <VerticalSpace space='34px' />
-
-        <ButtonContainer>
-          <NavButton
-            buttonType='primary'
-            text={getLocale('braveWalletWelcomeButton')}
-            onSubmit={() => history.push(WalletRoutes.OnboardingNewWalletTerms)}
-            maxHeight={'48px'}
-            minWidth={'267px'}
-          />
-
-          <NavButton
-            buttonType='secondary'
-            text={getLocale('braveWalletImportExistingWallet')}
-            onSubmit={() => history.push(WalletRoutes.OnboardingImportTerms)}
-            maxHeight={'48px'}
-            minWidth={'267px'}
-          />
-        </ButtonContainer>
-
-        <Row>
-          <SubDivider />
-          <SubDividerText>
-            {getLocale('braveWalletWelcomeDividerText')}
-          </SubDividerText>
-          <SubDivider />
-        </Row>
-
-        <NavButton
-          buttonType='primary'
-          text={getLocale('braveWalletConnectHardwareWallet')}
-          onSubmit={() =>
-            history.push(WalletRoutes.OnboardingHardwareWalletTerms)
-          }
-          maxHeight={'48px'}
-          minWidth={'267px'}
-        />
-
-        <VerticalSpace space='20px' />
-
-        <LearnMoreLink
-          href='https://support.brave.com/hc/en-us/categories/360001059151-Brave-Wallet'
-          target='_blank'
-          rel='noreferrer'
-        >
-          {getLocale('braveWalletLearnMoreAboutBraveWallet')}
-        </LearnMoreLink>
-      </OnboardingWrapper>
-    </WalletPageLayout>
+    <>
+      <WelcomePageBackground />
+      <WelcomePageWrapper>
+        <Content>
+          <Row
+            gap='10px'
+            justifyContent='flex-start'
+          >
+            <BraveIcon />
+            <Title>{getLocale('braveWalletTitle')}</Title>
+          </Row>
+          <VerticalSpace space={leo.spacing['5Xl']} />
```

Comment:
can you use `paddingBottom` on the row component instead of rendering a `VerticalSpace`?